### ISSUE TITLE
[Feature Request] disable end of word cursor snapping

### ISSUE THREAD

**Issue by FireNinja7365** [futo-org/android-keyboard#350]
Make an option to turn off the feature where the cursor snaps to the end of words when moved

**Comment by Tema726** [comment]
Yes please!

**Comment by DavidBevi** [comment]
Oh!, that's what was wrong yesterday when I was in a hurry.
Yes please, if I tap mid-word it's because I want the cursor mid-word!!!

**Comment by Timothy-Ecc19** [comment]
The ability to disable cursor auto snaping to the end of words would be Wonderful!

**Comment by JellyJoe198** [comment]
What is weird is it doesn't always move to the end, only after you've done certain actions which feel random if you're not used to them. See #267. The option to fully remove cursor snapping would solve both these issues.

**Comment by FireNinja7365** [comment]
Idk what i did to mine but somehow i accidentally disabled it so I'm happy if they fix it or not lol

**Comment by abb128** [comment]
Duplicate of #98
